soak/README.md

```diff
@@ -0,0 +1,202 @@
+## INTRODUCTION
+
+ACK team provides the soak testing framework for service teams where service teams can execute long running
+soak tests for testing the performance and stability of the service controllers.
+
+Under the hood, these soak tests consist of multiple end to end test runs for the service controller. End to end tests are present in service-controller github repository.
```

Comment:
nit: "consist of multiple end-to-end tests" I think we're implying here that "introduces load by repeatedly running the end-to-end tests"